entries/t/tvtime.com.json

```diff
@@ -1,15 +1,13 @@
 {
   "TV Time": {
     "domain": "tvtime.com",
-    "url": "https://www.tvtime.com",
-    "img": "tvtime.com.png",
     "contact": {
       "email": "support@tvtime.com",
       "facebook": "tvshowtime",
       "twitter": "tvshowtime"
     },
     "keywords": [
-      "entertainment"
+      "social"
```

Comment:
Why not both? We can't let the opportunity to use multiple keywords go to waste.

Reply:
I don't think it fits at all in the Entertainment category.